File: cpp/1047.cpp
#include <stdio.h>

using namespace std;

int main()
{
    int sh, sm, fh, fm; 
    scanf("%d %d %d %d", &sh, &sm, &fh, &fm);

    if (sh >= fh && sm >= fm) {
        fh += 24;
    }

    if (sm > fm) {
        fm += 60;
        fh--;
    }

    printf("O JOGO DUROU %d HORA(S) E %d MINUTO(S)\n", fh - sh, fm - sm);
    return 0;
}

File: py/1047.py
times = list(map(int, input().split()))

if times[2] <= times[0] and times[3] <= times[1]:
    times[2] = 24 + times[2]
    
if times[3] < times[1]:
    times[2] -= 1
    times[3] += 60
    
result = [times[2] - times[0], times[3] - times[1]]
    
print(f'O JOGO DUROU {result[0]} HORA(S) E {result[1]} MINUTO(S)')